hyper-schema.json

```diff
@@ -21,6 +21,8 @@
                     "type": "string",
                     "format": "uri-template"
                 },
+                "hrefPointers": {
+                    "description": "a map of variable names to relative instance JSON Pointers, which adjust the instance location from which template variable resolution begins",
```

Comment:
Missing closing `}`

Reply:
Also maybe a "type" property.

Reply:
I avoided a "type" on purpose.  While most will be objects, it would not be that odd to have an array.  I can't think of a situation where this sort of data would be best handled as a string or a number but I couldn't thing of a compelling reason to restrict it either.

We can always see how it gets used and tighten it in future drafts if it really seems necessary.  In general, the meta-schema errs on the side of permissiveness.